.travis.yml

```diff
@@ -19,3 +18,4 @@ otp_release:
   - 23.0
 services:
   - redis-server
+script: make check test
```

Comment:
^ Does `check` run reasonably fast? I've had bad experiences running Dialyzer on Travis CI in the past.

Reply:
Travis CI is activated for this project and builds take less than 3 min. for any given OTP version, so I'd say it's reasonable, sure.